apps/src/templates/sectionProgressV2/MoreDetailsDialog.jsx

```diff
@@ -0,0 +1,33 @@
+import PropTypes from 'prop-types';
+import React, {useState} from 'react';
+import i18n from '@cdo/locale';
+import Button from '@cdo/apps/templates/Button';
+import AccessibleDialog from '@cdo/apps/templates/AccessibleDialog';
+import {Heading3, Heading5} from '@cdo/apps/componentLibrary/typography';
+
+export default function MoreDetailsDialog({hasValidation, onClose}) {
+  return (
+    <AccessibleDialog onClose={onClose}>
```

Comment:
Note: Right now this is all place holder content.  I will be adding a lot more content in here in the future and hopefully cleaning up may of the divs.